samples/nrf9160/pdn_client/src/pdn_client.c

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <zephyr.h>
+#include <stdio.h>
+#include <string.h>
+#include <pdn.h>
+
+void main(void)
+{
+	printk("The AT host sample started\n");
+
+	char address[] = "default";
+
+	void *pdn_handle = pdn_open();
+	if (!pdn_handle) {
+		printk("Cannot create PDN handle!\n");
+		return;
+	}
+
+	if (pdn_connect(pdn_handle, address, strlen(address)) < 0) {
+		printk("Connection failed!\n");
+		pdn_close(&pdn_handle);
```

Comment:
It would possibly be enough to just log the error, we close the `pdn_handle` and exit the main thread right after.